### PR TITLE
[v0.34] Documentation for rw test's new --db-push flag

### PR DESCRIPTION
**Pull request by pachoclo**
Updates the CLI docs to include the work done in https://github.com/redwoodjs/redwood/pull/2642.

### Screenshot:  

Screen Shot 2021-05-29 at 4 16 32 PM

---
cc @dac09

### CODE REVIEW

File: docs/cliCommands.md [hunk 1]
```diff
@@ -1559,6 +1559,7 @@ yarn redwood test [side..]
 | `--watchAll`        | Run all tests                                                                                                                                                  |
 | `--collectCoverage` | Show test coverage summary and output info to `coverage` directory in project root. See this directory for an .html coverage report                            |
 | `--clearCache`      | Delete the Jest cache directory and exit without running tests                                                                                                 |
+| `--db-push`         | Syncs the test database with your Prisma schema without requiring a migration. It creates a test database if it doesn't already exist [default: true]. Note that this flag is ignored if your project does not have an api side. [👉 More details](https://redwoodjs.com/docs/cli-commands#prisma-db-push). |
```
Comment (jtoar) [inline docs/cliCommands.md:1562]: Just a few edits:
```suggestion
| `--db-push`         | Syncs the test database with your Prisma schema without requiring a migration. It creates a test database if it doesn't already exist [default: true]. This flag is ignored if your project doesn't have an `api` side. [👉 More details](https://redwoodjs.com/docs/cli-commands#prisma-db-push). |
```

**Review by jtoar — CHANGES_REQUESTED** [review]
@pachoclo thanks for updating the docs as well, really appreciate it! 👏  I made a few edits to the description.

Is it possible to only include the change you made to the cliCommands doc? Right now your PR includes changes to a few other files, like the `yarn.lock`.

Let me know if you need any help doing this!

**Comment by pachoclo** [comment]
> Is it possible to only include the change you made to the cliCommands doc? Right now your PR includes changes to a few other files, like the `yarn.lock`.

Sure sure. I was hesitant to commit them--I should have asked first :)

**Review by dac09 — APPROVED** [review]
HODL till next release. Thanks again @pachoclo

**Pull request merged**